tests/test_cli.py

```diff
@@ -35,17 +36,20 @@ def tmp_dir(tmp_path_factory) -> Path:
             "Fare": float
         }
         data_frame = pl.read_csv(csv_fp, dtypes=csv_dt)[:100]
-        meta_frame = MetaFrame.fit_dataframe(data_frame, spec={"PassengerId": {"unique": True}})
+        meta_frame = MetaFrame.fit_dataframe(data_frame, var_specs=[{"name": "PassengerId", "distribution": {"unique": True}}])
```

Comment:
Ah now I see why it changed to a list instead of a dict

Reply:
can we still do arbitrary order which does not align with the order of the columns in the data frame?

Reply:
Yes, if the column exists, the order doesn't matter. It only matters if the column is a data-free one.